docs/requirements-docs.txt

```diff
@@ -1,5 +1,5 @@
 opensearch-py>=2
-pandas>=1.5,<3
+pandas>=1.5.2,<2.1
```

Comment:
Did you get time to test functionalities with different versions of pandas? If yes, which versions did you test?

Reply:
Hey @dhrubo-os , I didn't complete yet. It's still in progress. I will let you know when it's done